SDPi_Supplement/asciidoc/volume2/gateways/tf2-ch-b-gateway-acm.adoc

```diff
@@ -1,5 +1,37 @@
-// = SDPi ACM Gateway -- Mapping
+=== SDPi ACM Gateway -- Mapping
 
 NOTE:  This file contains the SDPi gateway actor (SOMDS_ACM_Gateway) logic for mapping from SDC BICEPS MDIB content and interactions with IHE DEV "PCD" ACM profile messaging.
 
-=== Gateway:  IHE DEV ACM Profile
+==== Scope
+This chapter defines the mapping from SDC MDIB content as defined in this document and its underlying standards, to IHE Alert Communication Management (ACM) profile messages as defined in the <<ihe_pcd_tf-2_2019>>.
+
+The SDPi gateway actor represents the Alarm Reporter (AR) role of the IHE ACM profile.
+
+The following sections supplement the IHE ACM profile as appropriate. If there are no supplementing definitions, the definitions as described in the <<ihe_pcd_tf-2_2019>> will apply.
+
+==== Referenced Standards & Profiles
+This section provides an overview about the referenced standards and profiles used in this chapter:
+
+* <<ihe_pcd_tf-2_2019>>
+* <<ieee_11073_10702_202x>>
+* <<ieee_11073_10701_2022>>
+* <<ieee_11073_10700_2022>>
+* <<ieee_11073_10207_2017>>
+
+==== HL7 Segment Descriptions
+The following sections refer to the *Appendix B Common Segment Descriptions* of the <<ihe_pcd_tf-2_2019>>.
+
+include::tf2-ch-b-gateway-msh-mapping.adoc[]
+
+include::tf2-ch-b-gateway-pid-mapping.adoc[]
+
+include::tf2-ch-b-gateway-pv1-mapping.adoc[]
+
+===== OBR - Observation Request Segment
+The HL7 Observation Request (OBR) segment requires a mapping from the SDC containment tree and metric data to the OBR segment fields.
```

Comment:
```suggestion
The HL7 Observation Request (OBR) segment requires a mapping from the <<acronym_mdib>> containment tree and metric data to the OBR segment fields.
```

Reply:
Please ignore - ACM is not part of the review.